keyboards/mechlovin/infinity87/rev2/matrix.c

```diff
@@ -0,0 +1,289 @@
+/* Copyright 2020 Team Mechlovin'
```

Comment:
can this be converted to use matrix lite, instead of the full custom matrix? 

https://docs.qmk.fm/#/custom_matrix?id=lite